.env.example

```
@@ -1,6 +1,7 @@
 # PRIVATE_KEY            — Safe signer used for STAGING propose-* tasks (--environment=staging)
 # PRIVATE_KEY_PRODUCTION — Safe signer used for PRODUCTION propose-* tasks (default)
 PRIVATE_KEY="<your private key>"
+PRIV_KEY_PAUSER_WALLET_STAGING=  # pauser wallet key for staging diamonds (required by script/utils/testCheckPauserFunds.sh)
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)